Missing lodash dependency

```
kev@Kevs-MBP magica-voxel-parser $ node index.js
module.js:471
    throw err;
    ^

Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/kev/code/magica-voxel-parser/node_modules/riff-parser/riff/fmtChunk.js:6:14)
```

--- a/package.json
+++ b/package.json
@@ -23,6 +23,9 @@
     "url": "https://github.com/srveit/riff-js/issues"
   },
   "homepage": "https://github.com/srveit/riff-js/wiki",
+  "dependencies": {
+    "lodash": "^3.10.1"
+  },
   "devDependencies": {
     "gulp": "gulpjs/gulp#4.0",
     "gulp-istanbul": "^0.10.3",
@@ -37,7 +40,6 @@
     "jsdoc": "^3.3.1",
     "jshint": "^2.8.0",
     "jslint": "^0.9.1",
-    "lodash": "^3.10.1",
     "q": "^1.4.1"
   },
   "readme": "# riff-parser\nJavascript library for encoding and decoding RIFF format files.\n"

